website/server/controllers/api-v3/auth.js

```diff
@@ -231,7 +231,7 @@ api.loginLocal = {
 
 function _passportProfile (network, accessToken) {
   if (network in SUPPORTED_SOCIAL_NETWORKS) {
-    return new Promise.promisify((resolve, reject) => {
+    return new Bluebird((resolve, reject) => {
```

Comment:
That always results in `TypeError: A value undefined was yielded that could not be treated as a promise`